docs/test/configure-unit-tests-by-using-a-dot-runsettings-file.md

```diff
@@ -305,6 +314,27 @@ The video data collector captures a screen recording when tests are run. This re
 
 To customize any other type of diagnostic data adapters, use a [test settings file](../test/collect-diagnostic-information-using-test-settings.md).
 
+
+### Blame data collector
+
+```xml
+<DataCollector friendlyName="blame" enabled="True">
+  <Configuration>
+    <ResultsDirectory>C:\TestResults</ResultsDirectory>
+    <CollectDump CollectAlways="true" DumpType="mini" />
+  </Configuration>
+</DataCollector>
+```
+
+This option is helpful in isolating the problematic test causing test host crash. It creates an output file (*Sequence.xml*) in the directory specified by the ResultsDirectory node, which captures the order of execution of the test before the crash. 
```

Comment:
This setting would imply using blame collector to take a hang dump.

These are the settings that are equivalent to --blame:

```xml
<RunSettings>
    <LoggerRunSettings>
        <Loggers>
            <Logger friendlyName="blame" enabled="True" />
        </Loggers>
    </LoggerRunSettings>
    <DataCollectionRunSettings>
        <DataCollectors>
            <DataCollector friendlyName="blame" enabled="True">
            </DataCollector>
        </DataCollectors>
    </DataCollectionRunSettings>
</RunSettings>
```


The sequence file will end up in TestResults that folder that is automatically generated. The one mentioned in the settings is just a temp path.

Reply:
Is the role of blame option to be able to detect crashed tests and put them in Sequence.xml file? So what is difference with your example? About ResultsDirectory, we can specify folder where we want to get results, by default, as you said, it would create TestResults folder in working directory, so why do you call it 'temp path'?

Reply:
@nohwnd's feedback makes sense to me. Since the Sequence.xml's default location is the TestResults folder, we don't need to specify it in runsettings. This helps keep the example as simple as possible. (It is specified in other runsettings examples to show how to set a path, but we shouldn't include it in all the examples when it's not needed.)

I don't think we want the "golden path" use case to always collect a crash dump. I'd predict most users just want to know which test crashed the process.

Reply:
Got it. Example simplified.

Reply:
The TestResults path in the Blame collector settings is place where the dump file gets placed temporarily, and then it is moved to the real TestResults folder. It is confusing. New version of TP uses the local  temp folder.

You are right, the role of blame option **is** detecting the tests that crashed the test host. But the options you are showing are doing more, they enable taking a memory dump when the testhost process crashed.